src/endpoint/s3/ops/s3_head_bucket.js

```diff
@@ -1,13 +1,14 @@
 /* Copyright (C) 2016 NooBaa */
 'use strict';
 
+const s3_utils = require("../s3_utils");
+
 /**
  * http://docs.aws.amazon.com/AmazonS3/latest/API/RESTBucketHEAD.html
  */
-async function head_bucket(req) {
+async function head_bucket(req, res) {
+    s3_utils.set_response_supported_storage_classes(res);
     await req.object_sdk.read_bucket({ name: req.params.bucket });
```

Comment:
it would be better if this response was also responding based on the actual bucket namespace type so for namespace_fs buckets it would check config.NSFS_GLACIER_ENABLED, but for others it won't, which means that instead of setting this generically here, we should return the supported classes from read_bucket...
it might seem unneeded at the moment because currently this config is not used elsewhere, but when we do want to enable it for nsfs buckets in container deployments it would report the wrong classes for buckets.